### ISSUE TITLE
3.2.3. and 3.2.4. 2-Team Quizzes, 15 questions vs. 20 questions

### ISSUE THREAD

**Issue by jswingle** [gryphonshafer/Quizzing-Rule-Book#147]
The rulebook defines two styles of 2-team quizzes, one of which is designated as a "tiebreaker" quiz. 2-team quizzes are rarely used in my own quizzing circles, but sometimes are a scheduling necessity. I have seen the two different rulesets for 2-team quizzes confuse a quiz meet's logistics more than one time.

I don't think we need two separate rulesets for a rare occurrence. I would propose we cut the 20-question 2-team quiz and have all 2-team quizzes defined with the 15-question ruleset. I would go with the 15 questions because 20 is too many - not enough competition on jumps. But I care less about whether we do 15- or 20-, my main concern is making the rulebook less confusing.

**Comment by scottpeterson** [comment]
Yeah this is a weird case for sure. The stakes are low since I assume no one counts these tiebreaker quizzers for individual averages. But can't necessarily make that assumption. So probably want to create a tiebreaker quiz structure that sets up similar scoring incentives? And i think the 2-team, 15-question one does that decently. 

TLDR: I'd support removing the 2-team, 20-question quiz tiebreaker from the RB.

**Comment by kclimenhaga** [comment]
In principle, I agree with the idea to cut this down to one type of two-team quiz. However, what happens when a quiz is scheduled as a 3-team quiz, but one team does not show up? We have established that if a team shows up late, the quiz will start on time. In that case, it would have to be a regular 20-question quiz even if the third team never shows up.

**Comment by scottpeterson** [comment]
Yep, that scenario is just a normal 3 team 20 question quiz.   And one of the teams "never jumps" ;-)

**Comment by jswingle** [comment]
Yup, we would do it as a 3-team quiz anyways, to the extent that if two questions are erred on the third question is just eaten up with a pointless bonus. Although I wouldn't necessarily be opposed to a quiz meet director deciding to turn no-show quizzes into a 2-team 15-question quiz.

**Comment by jswingle** [comment]
My proposal for a language change here is pretty simple:

- Remove 3.2.4 from the rulebook entirely
- Change the title of 3.2.3 from "2-Team 15-Question Tie-Breaker Quiz" to just "2-Team 15-Question Quiz"
- Remove the bullet point under 3.2.3 which says "A quiz-out is 3 correct questions"

I don't think there is any need to differentiate different types of 2-team quizzes. If we need to do a tiebreaker quiz, have it be a 15-question quiz; likewise, if a meet's # of participating teams requires there be a 2-team quiz somewhere in the schedule, that quiz should just use the same rules as a 2-team tiebreaker quiz to make things simple.

I think a 2-team 20-question quiz makes the quiz way easier, and therefore it's an unfair advantage to the teams who happen to get placed in one of these quizzes during prelims. Reducing the # of questions to 15 would help balance this out. 

However, the current rule that quiz-outs are only 3 correct answers in a 15-question quiz, would make the individual statistic calculations VERY messy and frustrating. I think reverting to the default of 4 correct answers for a quiz out would be a fine and fair way to handle this problem. With 1 less team, quizzers going for a high individual score should still be able to achieve a quiz-out in 15 questions.

**Comment by levikoral** [comment]
I agree that sounds like a nice idea

**Comment by jswingle** [comment]
On behalf of the Rules Committee:

During our meeting on 9/12/2022, the Rules Committee voted 5-0 to keep this issue in discussion for now. This means we have neither rejected or approved this issue, but would like more input before we decide.

We agreed with the changes proposed above on my most recent post, but we realized that there should also be an added bullet point explaining what to do with type distributions in a 15-question quiz. 

Some members of the committee expressed a memory where 15-question quizzes used to be handled by the QM generating an ordinary 20-question quiz and reading out the types of the first 5 questions, with the quiz officially beginning on question #6. We looked at previous editions of the rulebook, but were unable to find this in any previous rulebook. It seems that the issue of "what do we do with type distributions in a 15-question quiz?" has _always_ been a rulebook bug. The "burn the first 5 questions and their types" strategy seems to have been a way of dealing with this bug.

In any case, we need to figure out a bullet point on type distributions before we push this change through.

For more information on the rulebook Change Management Process, please visit: https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/master/content/rule_book/change_management.md

**Comment by jswingle** [comment]
Though it may never have been a "real rule," I like the idea of just generating a 20-question quiz and burning 5 questions randomly. This is the simplest way to implement the rule, tech-wise.

I don't agree that QMs should simply "start from question 6" though. I think that's a confusing quizzer experience, and the rules specify bonus question and error points rules on 12-15 instead of the typical 16-20, so we want the question numbers to be correct.

**Comment by jswingle** [comment]
Adding entirely new distribution rules requires us to tweak them over time. It's a decent amount of work for almost no benefit. We shouldn't be encouraging 2-team quizzes to happen very often, if ever, anyways. It's OK if the rules are a little imperfect.

We could also remove exactly 1 of each type from the quiz, but this would make 2-team quizzes more likely on average to prioritize INT questions, and this would currently only work on epistle years (narrative years you'd have to randomly choose one type to NOT remove). We also currently have open proposals for changing all sorts of things related to distributions, such as splitting out CRs vs. CVRs, removing MAs from the competition entirely, and adding SIT variants to epistle years.

The simplest and most future-proof solution is to do 5 random burns.

**Comment by ARMediting** [comment]
The other consideration of what question to start on is scoresheets, and I wonder if this is why the unofficial rule began circulating. While the rulebook doesn't concern scoresheets, they aren't built to accommodate A's and B's at 12-15, and 2-team quizzes don't happen often enough to warrant a second sheet design.

Also, for anyone at Internationals this summer (@gryphonshafer @ZacharyTinker), how was the 2-team quiz there handled, both in distribution and numbering?2 Peter 1:19

**Comment by gryphonshafer** [comment]
CBQZ has "2-team 20-question" and "2-team 15-question" quiz setup settings, so in terms of QMing, I just ran whatever came up in CBQZ, which would have been numbered starting at 1, and attempting to fit minimums into the range, while not going over maximums, just like in a normal quiz.

I don't recall what my SK did, but I think she started on question 6 on the scoresheet and just renamed each question number on the question number row.

If a meet was planning to use paper scoresheets, it'd be trivial to add a duplicate secondary question numbers line that starts counting at column 6 with a label of question 1.

**Comment by jswingle** [comment]
However it gets worked out practically, I definitely think the rules should require using question numbers 1-15 with an announcement of which question types are being randomly removed from the distribution.

Starting at question 6 might simplify things from an official's perspective (which I do like) but I think it's off-putting to conduct an official quiz that way. (Not a great argument, I know.)

**Comment by kclimenhaga** [comment]
I think the reason we start from 6 is more for the sake of the quizzers. It's confusing to remember that error rules now apply to 12-15 instead of 16-20, so if we start from 6, the error rules fall on the same question numbers. Of course, the error rules would need to be on 17-20 instead of 16-20, but I think that's still less confusing for quizzers than having a quiz end on 15.
